packages/augur-test/src/flash/create-canned-markets-and-orders.ts

```diff
@@ -97,7 +97,7 @@ async function placeOrder(person: ContractAPI,
                           price: DecimalBigNumber) {
   const tickSize = can.tickSize
     ? new DecimalBigNumber(can.tickSize)
-    : numTicksToTickSize(new DecimalBigNumber("10000"), new DecimalBigNumber("0"), new DecimalBigNumber("0x0de0b6b3a7640000"));
+    : numTicksToTickSize(new DecimalBigNumber("100"), new DecimalBigNumber("0"), new DecimalBigNumber("0x0de0b6b3a7640000"));
```

Comment:
Will this bake in the 3 decimal places for price? Meaning traders and market makers will only trade prices 0.999. The same as 10th of a penny (really DAI)?